Documentation/API.md

```diff
@@ -0,0 +1,66 @@
+
+# API
+
+This addon exposes a small API to let you  
```

Comment:
Why have a newline here? Is there some sort of characters per line limit we are adhering to? Or maybe some documentation browser inside of FreeCAD that I am unaware of?

Reply:
Not sure what line you mean specifically?

I assume you mean the last one in the paragraph?

GitHub markdown adds `<br/>` tags for any double
space at the end of a line, and since we don't have  
CSS's `text-wrap : balance`, I prefer to do it that way.

There currently isn't a proper limit, which is the
problem as written text should be at most 66ch 
per line, depending on the content I break earlier
to balance the strings per line.